content/en/dora_metrics/_index.md

```diff
@@ -279,7 +279,29 @@ Events can be sent both at the start of and after incident resolution. Incident
 
 You can access and visualize your DORA metrics and filter them by team, service, repository, environment, and time period on the [DORA Metrics page][8].
 
-Use the information on this page to identify improvements or regressions for each metric, visualize changes, and compare trends over time. DORA metrics can be exported to dashboards and alerted on using [metric monitors][9].
+Use the information on this page to identify improvements or regressions for each metric, visualize changes, and compare trends over time. DORA metrics can be exported to dashboards or notebooks and be alerted on using [metric monitors][9].
+
+The metrics can also be queried with the API endpoints [Query timeseries points][10] and [Query timeseries data across multiple products][11].
+
+The metrics provided by DORA Metrics are:
+- `dora.deployments.count` of type `count`. Used for Deployment Frequency.
+- `dora.change_lead_time` of type `distribution`. Contains the age in `seconds` of the git commits at the time of deployment.
```

Comment:
Do we have a different page describing exactly how this is computed? If we do, I would reference that. If we don't, I would expect customers to ask about this in future